sonar-update-center-mojo/src/main/java/org/sonar/updatecenter/mojo/CompatibilityMatrix.java

```diff
@@ -108,13 +114,11 @@ public static class Plugin {
     private final Map<String, String> compatibleVersionBySqVersion = new HashMap<>();
```

Comment:
It's out of scope, but I've just noticed that homepageUrl is never used; it would be nice to clean it up :)

Reply:
done 👍